README.md

```diff
@@ -13,26 +15,13 @@ In the matrix-based security administrator can make reference to:
  -  any valid Tuleap user
  -  any user group defined in any project with the syntax `Tuleap project short name#user group name` (example: `my_project#project_members`)
 
-For more information about the integration between Tuleap and Jenkins see: https://docs.tuleap.org/user-guide/ci.html?plugins-configuration#continuous-integration
-
-## Installation
-You can either install the plugin via the [Jenkins Plugins Marketplace](https://www.jenkins.io/doc/book/managing/plugins/#from-the-web-ui), or manually.
-If you install the plugin manually you have to:
- - Clone the repository
- - In the repository directory: ``` mvn clean install ```
- - See https://www.jenkins.io/doc/book/managing/plugins/#advanced-installation
-If you want to install via the Jenkins CLI see: https://www.jenkins.io/doc/book/managing/plugins/#advanced-installation
```

Comment:
Maybe we should also keep this part for the devs who want to work on that plugin locally.

What do you think?